src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -170,22 +148,15 @@ public class WalletManager : IWalletManager
             WalletSettings walletSettings,
             DataFolder dataFolder,
             IWalletFeePolicy walletFeePolicy,
-            IAsyncProvider asyncProvider,
-            INodeLifetime nodeLifetime,
             IDateTimeProvider dateTimeProvider,
-            IScriptAddressReader scriptAddressReader,
-            IWalletRepository walletRepository,
-            IBroadcasterManager broadcasterManager = null) // no need to know about transactions the node will broadcast to.
+            IWalletRepository walletRepository) // no need to know about transactions the node will broadcast to.
```

Comment:
Seems that this comment possibly related to the `broadcasterManager`?

Reply:
Ah yes sure, I will remove it :)